buku.py

```diff
@@ -1588,18 +1582,8 @@ def print_rec(self, index=0, low=0, high=0, is_range=False):
             if self.field_filter == 0:
                 for row in resultset:
                     print_single_rec(row)
```

Comment:
Call `print_single_rec` within `print_rec_with_filter` and replace calls to `print_single_rec` everywhere else with `print_rec_with_filter`.

Make the default field filter value 0 so you can omit the param when the intention is to call `print_single_rec` or filter is not relevant.

Reply:
Okay sounds good. There's one place where `print_single_rec` is called where I'm not sure if it should be called from `print_rec_with_filter`:
https://github.com/jarun/Buku/blob/4a817e583ce4908aa61eafdb7eaffadb2522cc9a/buku.py#L3059-L3065. within the prompt, where it's passed an index.

I could add another optional keyword argument to `print_rec_with_filter` that could then be passed to `print_single_rec` but this seems a bit roundabout. Please let me know.